mapper/orm-coordination-database-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/databasepolling/cfg/HibernateOrmMapperDatabasePollingSettings.java

```diff
@@ -178,6 +198,7 @@ private Defaults() {
 		public static final boolean COORDINATION_PROCESSORS_INDEXING_ENABLED = true;
 		public static final int COORDINATION_PROCESSORS_INDEXING_POLLING_INTERVAL = 100;
 		public static final int COORDINATION_PROCESSORS_INDEXING_BATCH_SIZE = 50;
+		public static final Integer COORDINATION_PROCESSORS_INDEXING_TRANSACTION_TIMEOUT = null;
```

Comment:
That's not really a default, that's the absence of default. Could you avoid exposing this, and just use an `orElse( null )` when you retrieve the value of the configuration property?